Workflow/Sources/SubtreeManager.swift

```diff
@@ -194,7 +202,7 @@ extension WorkflowNode.SubtreeManager {
             let eventPipe = EventPipe()
             eventPipes.append(eventPipe)
 
-            /// See if we can
+            /// See if we can reuse an existing child node for the given key.
```

Comment:
this truncated sentence has bothered me for a long time – i think this is probably what it was going to